core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheFileInStreamTest.java

```diff
@@ -71,50 +73,140 @@
       (int) sConf.getBytes(PropertyKey.USER_CLIENT_CACHE_PAGE_SIZE);
 
   @Test
-  public void readPageCacheMiss() throws Exception {
-    Map<AlluxioURI, byte[]> files = new HashMap<>();
-    AlluxioURI testFilename = new AlluxioURI("/test");
+  public void readFullPage() throws Exception {
     int fileSize = PAGE_SIZE;
     byte[] testData = generateData(fileSize);
```

Comment:
nit: how about using `ByteUtils.getIncreasingByteArray`?

Reply:
Oh yes, I was looking for that